libraries/ms-drivers/src/ltc_afe_impl.c

```diff
@@ -104,6 +103,41 @@ static StatusCode prv_trigger_aux_adc_conversion(LtcAfeStorage *afe) {
   return spi_exchange(settings->spi_port, cmd, 4, NULL, 0);
 }
 
+static StatusCode prv_mux_write_comm_register(LtcAfeStorage *afe, uint8_t device_cell) {
+  if (device_cell >= AUX_ADG731_NUM_PINS) {
+    return STATUS_CODE_OUT_OF_RANGE;
+  }
+  LtcAfeSettings *settings = &afe->settings;
+  LtcAfeWriteCommRegPacket packet = { 0 };
+  // Build WRCOMM Command
+  prv_build_cmd(LTC6811_WRCOMM_RESERVED, packet.wrcomm, SIZEOF_ARRAY(packet.wrcomm));
+  // Write 3 bytes of data to the COMM registers
+  // We send the a byte and then we send CSBM_HIGH to
+  // release the SPI port
+  packet.reg.icom0 = LTC6811_ICOM_CSBM_LOW;
+  packet.reg.d0 = device_cell;
+  packet.reg.fcom0 = LTC6811_FCOM_CSBM_HIGH;
+  uint16_t comm_pec = crc15_calculate((uint8_t *)&packet.reg, sizeof(LtcAfeCommRegisterData) / 3);
```

Comment:
From the datasheet:
> Similar to the I2C master, if ICOMn[3:0] specified a CSBM HIGH or a NO TRANSMIT condition, the CSBM, SCKM and SDIOM lines of the SPI master are released and the rest of the data in the word is ignored.

You def. don't want to leave icom as all zeros because that'll configure it to use i2c and then send stuff you don't want. If you don't want to send data for the following 2 bytes then either set it the icon fields to CSBM HIGH or NO TRANSMIT. I'd prefer the later because it's a bit more explicit

You also want to calculate pec on the whole register, otherwise the chip will get a different pec result ...